site.yml

```diff
@@ -1,4 +1,48 @@
 ---
-- include: redhat_release.yml
-- include: autotested.yml
-- include: cleanup.yml
+
+# Setup/override global variables
+- hosts: all
+  tasks:
+    # Defines adept_job_no_group and rt_adept_job_no
+    - include: roles/common/tasks/adept_job_no.yml
+
```

Comment:
Argh, actually might need to do something like this.  I just predicted and proved there are problems at the top-level if the variable isn't available at the playbook scope.  Uggg.   Still, it's two-steps forward, and only one back :smile:   I'll figure something out.